docs/pages/index.mdx

```diff
@@ -44,11 +44,11 @@ MUD comes with:
 2. **World**: An [entry-point kernel](world) that brings standardized access-control, upgrades, and modules.
 3. Blazing fast development tools based on [Foundry](https://github.com/foundry-rs/foundry)
 4. Client-side data-stores that magically reflect onchain state. No need to use view functions and events to get your contract data.
-5. **MODE**: [A Postgres database](mode) you can query with SQL that reflects your onchain state 1 to 1. Never write an indexer again.
+5. **Indexer**: [A indexer](indexer) you can query with [tRPC](https://trpc.io/docs) that reflects your onchain state 1 to 1.
 
 The projects built with MUD speak for themselves: some of the most complex applications on Ethereum have been built with the framework in record time. OPCraft, a fully onchain voxel world, was built by Lattice in 1.5 months. While it didn’t have the security and auditing requirement of financial applications, it handled more transaction throughput than most apps on Mainnet ever did in their lifetime: in 10 days, OPCraft players made 3.5 million transactions, filling the MUD onchain database with billions of gas worth of storage; with the client seamlessly handling synchronizing that state back. OPCraft used MUD without any additional tricks: large onchain applications with minimal client headaches can be yours too!
 
-MUD is maximally onchain: the entire application state lives in the EVM, and the only requirement for clients and frontends is an Ethereum Node or a MODE — a MUD node. You can use all MUD libraries together as a framework and get super-powers, or only pick the parts you like. It’s up to you!
+MUD is maximally onchain: the entire application state lives in the EVM, and the only requirement for clients and frontends is an Ethereum Node or an Indexer — a MUD node. You can use all MUD libraries together as a framework and get super-powers, or only pick the parts you like. It’s up to you!
```

Comment:
```suggestion
MUD is maximally onchain: the entire application state lives in the EVM, and the only requirement for clients and frontends is an Ethereum Node or [an indexer](indexer). You can use all MUD libraries together as a framework and get super-powers, or only pick the parts you like. It’s up to you!
```

MODE was short for MUD Node and we had plans to add more features to the MODE umbrella that made it more of a "node" than just an indexer. For now there are no such plans for the new indexer and instead we're focussing on small services/packages that do one thing well.